ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -2453,110 +2252,4 @@ describe(RuntimeConfigurationPanel.name, () => {
       spinDiskElement('num-preemptible');
     expect(preemptibleCountInput.disabled).toBeTruthy();
   });
-
-  // These tests require a little more simplification, so we are skipping them for now. However, they will run as
-  // part of runtime-configuration-panel.enzyme.spec.tsx.
-  test.skip.each([
-    [
-      'disk type',
-      [pickSsdType],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-    [
-      'disk decrease',
-      [decrementDetachableDiskSize],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-    [
-      'in-place + disk type',
-      [changeMainCpu_To8, pickSsdType],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-    [
-      'in-place + disk decrease',
-      [changeMainCpu_To8, decrementDetachableDiskSize],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-    ['recreate', [clickEnableGpu], { wantDeleteRuntime: true }],
-    [
-      'recreate + disk type',
-      [clickEnableGpu, pickSsdType],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-    [
-      'recreate + disk decrease',
-      [clickEnableGpu, decrementDetachableDiskSize],
-      { wantDeleteDisk: true, wantDeleteRuntime: true },
-    ],
-  ] as DetachableDiskCase[])(
-    'should allow runtime to recreate to attached PD: %s',
-    async (desc: string, setters, expectations) => {
-      setCurrentRuntime(detachableDiskRuntime());
-
-      const disk = existingDisk();
-      setCurrentDisk(disk);
-
-      const { container } = component();
-      await runDetachableDiskCase(
-        container,
-        [desc, setters, expectations],
-        disk.name
-      );
-    }
-  );
-
-  test.each([
-    ['disk increase', [incrementDetachableDiskSize]],
-    ['in-place', [changeMainCpu_To8]],
-    [
-      'in-place + disk increase',
-      [changeMainCpu_To8, incrementDetachableDiskSize],
-    ],
-    ['recreate + disk increase', [clickEnableGpu, incrementDetachableDiskSize]],
-  ])(
-    'should allow runtime updates to attached PD: %s',
-    async (desc: string, setters) => {
-      setCurrentRuntime(detachableDiskRuntime());
-
-      const disk = existingDisk();
-      setCurrentDisk(disk);
-
-      const { container } = component();
-      const updateRuntimeSpy = jest.spyOn(runtimeApi(), 'updateRuntime');
-
-      for (const action of setters) {
-        await action(container);
-      }
-
-      clickButtonIfVisible('Next');
-      clickButtonIfVisible('Update');
-      await waitFor(() => {
-        expect(updateRuntimeSpy).toHaveBeenCalledTimes(1);
-      });
-      expect(disksApiStub.disk.name).toEqual(disk.name);
-    }
-  );
-
-  test.each([
-    ['disk type', [pickSsdType], { wantDeleteDisk: true }],
-    ['disk decrease', [decrementDetachableDiskSize], { wantDeleteDisk: true }],
-    ['disk increase', [incrementDetachableDiskSize], { wantUpdateDisk: true }],
-  ] as DetachableDiskCase[])(
-    'should allow runtime creates with existing disk: %s',
```

Comment:
need equivalent **create** runtime (current = null) tests for:
* disk type change -> delete disk 
* disk increase -> don't delete disk
* disk decrease -> delete disk

Reply:
Replaced with:
'should require disk deletion when attempting to create a runtime with a different disk type'
'should not delete disk when attempting to create a runtime with a larger disk'
'should require disk deletion when attempting to create a runtime with a smaller disk'

It seems unclear as to how we would end up in a situation where the runtimeStore has a null value for runtime and we have a detached disk. It seems more likely that we would have a 'DELETED' runtime in the store and a detached disk like shown in: 'should reattach to an existing disk by default, for deleted VMs'